python/lsst/ts/dsm/utils.py

```diff
@@ -0,0 +1,75 @@
+import csv
+import os
+import yaml
+
+from astropy.time import Time, TimeDelta
+import numpy as np
+import pytz
+
+__all__ = ['convert_time', 'create_telemetry_config', 'create_telemetry_data']
+
+
+def convert_time(in_time):
+    """Convert an ISO UTC timestring to TAI timestamp.
+
+    Parameters
+    ----------
+    in_time : str
+        The time to convert.
+
+    Returns
+    -------
+    float
+        The TAI time corresponding to the input.
+    """
+    ptime = Time(in_time, scale='utc')
+    # replace allows this to work if clock isn't in UTC.
+    return ptime.tai.datetime.replace(tzinfo=pytz.utc).timestamp()
```

Comment:
The dome seeing monitor UI, which generates the telemetry, is being developing in coordination with CTIO and others. I was trying to keep things a bit agnostic in case they wanted to use the telemetry files. I think I will change the telemetry file format to insert UNIX timestamps so that I can then just leverage your `salobj` call.